DDDEastAnglia.Tests/Controllers/HomeControllerBuilder.cs

```diff
@@ -0,0 +1,28 @@
+using DDDEastAnglia.Controllers;
+using DDDEastAnglia.DataAccess;
+
+namespace DDDEastAnglia.Tests.Controllers
+{
+    public class HomeControllerBuilder
+    {
+        private IConferenceLoader conferenceLoader = new ConferenceLoaderBuilder().Build();
```

Comment:
Justification for this:

It uses the builders to create the conferenceloader so that when someone changes the dependencies of the conferenceloader you don't have loads of constructors to fix.

This is what happened to me when I added a dependency to the HomeController.. I had 15 constructors to go and fix.. 

Not too fussed though as I imagine conferenceloader won't change too much